src/components/city.js

```diff
@@ -0,0 +1,64 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink, useLocation } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import { fetchCities } from '../Redux/Reducers/Cities';
+import '../App.css';
+
+const Cities = () => {
+  const dispatch = useDispatch();
+  const location = useLocation();
+  const countryInfo = location.state.info;
+  useEffect(() => {
+    dispatch(fetchCities(countryInfo.name));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
As mentioned earlier and as requested by the previous reviewer, kindly avoid disabling linter checks. You may read more about the [react-hooks/exhaustive-deps](https://blog.logrocket.com/understanding-react-exhaustive-deps-linting-warning/).

Reply:
thanks please can you help me out how do I disable it without having errors

Reply:
I have spent hours with other partners trying to solve this particular issue but we could not please